src/main/java/proj/kedabra/billsnap/presentation/controllers/NotificationController.java

```diff
@@ -50,6 +51,7 @@ public NotificationController(final NotificationFacade notificationFacade, final
     @ApiResponse(responseCode = "404", content = @Content(schema = @Schema(implementation = ApiError.class)), description = "The invitation does not exist.")
     @ApiResponse(responseCode = "405", content = @Content(schema = @Schema(implementation = ApiError.class)), description = "The bill is not in Open status.")
     @ResponseStatus(HttpStatus.OK)
+    @PreAuthorize("hasAuthority('INVITATION_' + #invitationId)")
```

Comment:
Woah careful here, in develop we use billId not notification ID. I'd brease and recheck this. Everything is based around billId now. Honestly, you could probably get away with just using #billId and you'll be fine

Reply:
I think I branched out without fetching develop branch lol

Reply:
doesn't look like this was changed yet. It's billId in develop, are you rebasing origin/develop?